Buildings/Controls/OBC/FDE/DOAS/SupplyFanController.mo

```diff
@@ -0,0 +1,179 @@
+within Buildings.Controls.OBC.FDE.DOAS;
+block SupplyFanController "This block manages start, stop, status, and speed of the supply fan."
+
+  parameter Real yMinDamSet(
+  min = 0,
+  final unit = "Pa",
+  final quantity = "PressureDifference") = 125
+  "Minimum down duct static pressure reset value" annotation(Dialog(group = "DDSP range"));
+
+  parameter Real yMaxDamSet(
+  min = 0,
+  final unit = "Pa",
+  final quantity = "PressureDifference") = 500
+  "Maximum down duct static pressure reset value" annotation(Dialog(group = "DDSP range"));
+
+  parameter Real kPDucSetCV(
+  min = 0,
+  final unit = "Pa",
+  final quantity = "PressureDifference") = 250 "Constant volume down duct static pressure set point";
+
+  parameter Real damSet(
+  min = 0,
+  max = 1,
+  final unit = "1") = 0.9
+  "DDSP terminal damper percent open set point";
+
+  parameter Boolean vvUnit = true
```

Comment:
Change to is_vav. "True: System has zone terminals with variable damper position. False: System has zone terminals with constant damper position."